### PR TITLE
COMMERCE-2115 fixed checked payment method

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test

File: commerce-checkout-web/src/main/resources/META-INF/resources/checkout_step/payment_method.jsp [hunk 1]
```diff
@@ -56,7 +56,7 @@ String commercePaymentMethodKey = BeanParamUtil.getString(commerceOrder, request
 
 					<li class="commerce-payment-types list-group-item list-group-item-flex">
 						<div class="autofit-col autofit-col-expand">
-							<aui:input checked="<%= commercePaymentMethod.getKey() == commercePaymentMethodKey %>" label="<%= commercePaymentMethod.getName(locale) %>" name="commercePaymentMethodKey" type="radio" value="<%= commercePaymentMethod.getKey() %>" />
+							<aui:input checked="<%= commercePaymentMethod.getKey().equals(commercePaymentMethodKey) %>" label="<%= commercePaymentMethod.getName(locale) %>" name="commercePaymentMethodKey" type="radio" value="<%= commercePaymentMethod.getKey() %>" />
```
Comment (marco-leo) [inline commerce-checkout-web/src/main/resources/META-INF/resources/checkout_step/payment_method.jsp]: @andrea-ale-sbarra avoid chain, better do commercePaymentMethodKey.equals(commercePaymentMethod.getKey())

**Review by marco-leo — CHANGES_REQUESTED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:test

File: commerce-checkout-web/src/main/resources/META-INF/resources/checkout_step/payment_method.jsp [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (ethanbustad) [inline commerce-checkout-web/src/main/resources/META-INF/resources/checkout_step/payment_method.jsp:1]: @andrea-ale-sbarra looks like an extra line got in here -- can you remove in a follow-up PR?

**Pull request merged**